routes/projects/files/get-one.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * GET /projects/:username/:projectName/files/:pathToFile
+ * The :pathToFile parameter must be URI encoded
+ * Returns the contents of the given file in the form:
+ * {
+ *   type: <string> content-type,
+ *   path: <string>,
+ *   size: <integer>,
+ *   link: <string>
```

Comment:
It looks like we're leaning towards postgres. I think it would be simplest to just return the buffer here.